apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -29,10 +32,41 @@ const useSections = () => {
   return [sections, updateSection];
 };
 
+const saveSection = (e, section) => {
+  e.preventDefault();
+
+  const form = document.querySelector(`#${FORM_ID}`);
+  if (!form.checkValidity()) {
+    form.reportValidity();
+    return;
+  }
+
+  const csrfToken = document.querySelector('meta[name="csrf-token"]')
+    .attributes['content'].value;
+
+  fetch(SECTIONS_API, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'X-CSRF-Token': csrfToken
+    },
+    body: JSON.stringify({section})
+  })
+    .then(response => response.json())
+    .then(data => {
+      // Redirect to the sections list.
+      window.location.href = window.location.origin + '/home';
```

Comment:
this hardcoded redirect seems a little strange to me, is there a better way?

Reply:
We do have a [helper function](https://github.com/code-dot-org/code-dot-org/blob/7439ee06fc497031bb0ba2b0f4c2b6621e3f2a32/apps/src/utils.js#L754) for this but it just does the same thing you're doing. It can be useful for testing but otherwise I _think_ this is the best way